agasc/agasc.py

```diff
@@ -353,4 +361,42 @@ def get_stars(ids, agasc_file=None, dates=None, fix_color1=True):
         update_color1_column(t)
     t['DATE'] = dates
 
+    t = _add_supplement(t, supplement_file)
+
     return t if ids.shape else t[0]
+
+
+def _add_supplement(table, supplement_file=None):
+    """
+    Add information from AGASC supplement.
+
+    :param table: astropy.table.Table
+    :param supplement_file: str.
```

Comment:
Using a `LazyDict` will not allow having this selectable for each query, but for specialized cases the user can always override `DEFAULT_SUPPLEMENT_FILE` in the module before doing the first query.